src/components/AI/Cards.tsx

```diff
@@ -154,25 +155,13 @@ export default function Cards({
   };
 
   return (
-    <Box
+    <CustomSlider
```

Comment:
This works well, except naming, cause this isnt just slider, but wrapper layout with styled scroll bar, also this naming could be messed with `<Slider/>` component from mui